### DISCUSSION TITLE
Vercel deploys

### DISCUSSION THREAD

**Discussion by Mir4nda** [jeffreylanters/react-unity-webgl#444] [Help Wanted!]
Is it possible to deploy webgl in vercel ? I try to test but I receive this message:

_**index.d2474201.js:181 
           
       GET https://treatment-planner6.vercel.app/build/public.loader.js net::ERR_ABORTED 404**_

**Comment by LiquidityPanic** [comment]
You can just upload your folders directly from the Unity WebGL build into GitHub and run on Vercel - this is only needed if you are incorporating it into an existing react app. 

Previously hosted my game on Vercel and game here to integrate into my site - cheers.
